### PR TITLE
docs (v6.1.0)

### PR DESCRIPTION
**Pull request by Faf4a**
1. changed every
    - yes -> true
    - no -> false
2. renamed server -> guild
3. fixed last missing things

### CODE REVIEW

File: src/introduction/introduction.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ slug: /
 
 ## Installation
 
-**node.js 16.9.0 or newer is required.**
+**node.js 17.0.0 or newer is required.**
```
Comment (Leref) [inline src/introduction/introduction.md:43]: 16.9.0 required.

File: src/guides/1setup.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ id: setup
 
 ## Installation
 
-**node.js 16.9.0 or newer is required.**
+**node.js 17.0.0 or newer is required.**
```
Comment (Leref) [inline src/guides/1setup.md:9]: 16.9.0 required.

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/guides/1setup.md (hunk 2 above)
Comment (Faf4a) [reply src/guides/1setup.md]: fixed

File: src/introduction/introduction.md (hunk 1 above)
Comment (Faf4a) [reply src/introduction/introduction.md]: fixed

**Review by Leref — APPROVED** [review]

**Pull request merged**